.github/workflows/pull_request.yml

```
@@ -39,7 +39,7 @@ jobs:
     runs-on: linux-amd64-cpu4
     timeout-minutes: 60
     container:
-      image: rapidsai/ci:cuda11.5.1-ubuntu20.04-py3.8
+      image: rapidsai/ci-conda:cuda11.5.1-ubuntu20.04-py3.8
```

```suggestion
      image: rapidsai/ci-conda:latest
```

It looks like this particular image tag is no longer being published.

We should switch to using the `latest` alias here if this job doesn't depend on any particular CUDA/OS/Python versions.

Thoughts, @dagardner-nv?